# This patch file was generated by NetBeans IDE
# This patch can be applied using context Tools: Apply Diff Patch action on respective folder.
# It uses platform neutral UTF-8 encoding.
# Above lines and this line are ignored by the patching process.
--- views_handler_filter_term_node_tid_depth_limited_1.inc
+++ views_handler_filter_term_node_tid_depth_limited.inc
@@ -55,6 +55,12 @@
       '#default_value' => $this->options['min_depth_limit_for_list'],
       '#description' => t('This will limit the exposed version of this filter to showing terms either above or below the specified depth.'),
     );
+    $form['expose_children_of'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Expose select list to limit overall depth'),
+      '#default_value' => $this->options['expose_children_of'],
+      '#description' => t('If your filter is exposed, checking this will also expose the select list to limit nodes to children of the selected term.'),
+    );
   }
 
 
@@ -203,7 +209,7 @@
         '#size' => min(9, count($options)),
         '#default_value' => $default_value,
       );
-      
+      if ($this->options['expose_children_of']) {
       $form['children_of'] = array(
         '#title' => t('Limit exposed dropdown to children of term'),
         '#type' => 'select',
@@ -211,6 +217,7 @@
         '#options' => array(0 => '<Show all>') + $all_terms,
         '#default_value' => $this->options['children_of'],
       );
+      }
 
       if (!empty($form_state['exposed']) && !isset($form_state['input'][$identifier])) {
         $form_state['input'][$identifier] = $default_value;
